rest-adapter/src/main/java/uk/gov/justice/services/adapter/rest/envelope/RandomUUIDGenerator.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.services.adapter.rest.envelope;
+
+import java.util.UUID;
+
+/**
+ * Factory for creating new UUIDs.
+ */
+public class RandomUUIDGenerator {
+
+    /**
+     * Generate a random UUID.
+     * @return a UUID
+     */
+    public UUID generate() {
+        return UUID.randomUUID();
```

Comment:
This seems like a slightly elaborate wrapper...? I presume that this is to make something testable via mocking? If so, could we change the responsibilities somewhere to accept a UUID rather than generating one, to simplify?